README.md

```diff
@@ -17,51 +17,53 @@ To contribute:
-| [Rocket Lab](https://boards.greenhouse.io/rocketlab/jobs/5679154003)                                                               | Toronto, US locations                   | Unknown               | Summer 2024 Internship/ Co-Op - Software                                                       | Aug 2       |
-| [Cisco](https://jobs.cisco.com/jobs/ProjectDetail/Software-Engineer-Intern-Canada/1405428?uid=WdGd8tgOdAzFLoUI)                    | Ottawa                                  | Rolling Applications  | Software Engineer (Intern) Canada                                                              | Jul 10      |
-| Intuit                                                                                                                             | Toronto                                 | :lock: Closed :lock:  | Software Developer Co-op - 8 Months                                                            | Jun 27      |
-| Google                                                                                                                             | Toronto, Waterloo                       | :lock: Closed :lock:  | Software Developer Intern, BS, Summer 2024                                                     | Jun 6       |
+| Name                                                                                                                                                                                                                                                                                                                                        | Location                                | Application Period    | Notes                                                                                          | Date Posted |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------- | --------------------- | ---------------------------------------------------------------------------------------------- | ----------- |
+| [Google](https://www.google.com/about/careers/applications/jobs/results/88626154719584966-step-intern-secondyear-student-summer-2024?src=Online%2FGoogle%20Website%2FByF&utm_source=Online%20&utm_medium=careers_site%20&utm_campaign=ByF&company=Fitbit&company=Google&company=YouTube&distance=50&employment_type=INTERN&location=Canada) | Waterloo, Montreal, Toronto             | Unknown               | STEP Intern, Second-Year Student                                                               | Oct 2nd     |
```

Comment:
Maybe shorten the URL to just `https://www.google.com/about/careers/applications/jobs/results/88626154719584966-step-intern-secondyear-student-summer-2024`

Reply:
Thanks, done!